bip-0155.mediawiki

```diff
@@ -134,7 +134,7 @@ See the appendices for the address encodings to be used for the various networks
 
 Introduce a new message type <code>sendaddrv2</code>. Sending such a message indicates that a node can understand and prefers to receive <code>addrv2</code> messages instead of <code>addr</code> messages. I.e. "Send me addrv2".
 
-<code>sendaddrv2</code> SHOULD be sent after receiving the <code>verack</code> message from the peer.
+The <code>sendaddrv2</code> message MUST be sent in response to the <code>version</code> message from a peer and prior to sending the <code>verack</code> message.
```

Comment:
I think this would be a bit clearer as s/MUST be sent/MUST only be sent/ to indicate that the message may only be sent during a specific window rather than that the message must be sent by any nodes that comply with BIP155.  Without the addition of "only", I think you could argue that https://github.com/bitcoin/bitcoin/pull/20564 would be a violation of the spec since it implements BIP155 but won't send `sendaddrv2` to pre-70016 nodes.

Otherwise, this LGTM.

Reply:
Done